src/randoop/FailureSet.java

```diff
@@ -5,17 +5,27 @@
 import java.util.List;
 import java.util.Set;
 
+import randoop.operation.MethodCall;
+import randoop.sequence.ExecutableSequence;
+import randoop.sequence.MutableSequence;
+import randoop.sequence.MutableVariable;
+import randoop.sequence.Statement;
+import randoop.sequence.Variable;
+
 public class FailureSet {
   
   private Set<Failure> failures = new LinkedHashSet<Failure>();
   
   public static class Failure {
-    public final StatementKind st;
+    
+    public final Statement st;
     public final Class<?> viocls;
-    public Failure(StatementKind st, Class<?> viocls) {
-      this.st = st;
+    
+    public Failure(Statement st2, Class<?> viocls) {
```

Comment:
Please don't make this change.  It's much clearer to use the same variable name for the constructor formal parameter and the field name (and doing so is standard practice).